src/host/proxy/Host.Proxy.vcxproj

```diff
@@ -7,6 +7,7 @@
     <ProjectName>OpenConsoleProxy</ProjectName>
     <TargetName>OpenConsoleProxy</TargetName>
     <ConfigurationType>DynamicLibrary</ConfigurationType>
+    <ConfigurationType Condition="'$(Configuration)'=='Fuzzing'">StaticLibrary</ConfigurationType>

```

Comment:
WHAT

this can't be right, right? When we're building the solution for fuzzing, we need to compile the defterm handoff proxy as a `/lib`‽‽

Reply:
Yeah uh what.

Reply:
I'll need @DHowett 's input here. Without it, I get this error:

```
4>LINK : fatal error LNK1394: cannot infer default ASAN libraries; please specify them on the link command line
```

Reply:
So, this is necessary because:

1. OpenConsoleProxy is a hard build dependency of OpenConsole, because it produces the interface headers for its IDL files.
2. OpenConsoleProxy builds a DLL because it has to be shipped to end users to contain the proxy definitions.
3. When you try to build with Fuzzing, however, that DLL build fails. It's not a fuzzing target, but it tried to produce a usable output PE file. _Oops._

We switch the type to static library to make sure [1] still works, without triggering [3], and it is safe because nobody is consuming the proxy dll here (invalidating [2]).

Reply:
Please put this in a comment in the vcxproj file, because this is MAD

Reply:
(blocking on this)